phoenix-core/src/it/java/org/apache/phoenix/end2end/index/ImmutableIndexIT.java

```diff
@@ -417,6 +422,58 @@ public void testGlobalImmutableIndexDelete() throws Exception {
         }
     }
 
+    @Test
+    public void testGlobalImmutableIndexFullWriteOnLast() throws Exception {
```

Comment:
Does the existing test case cover "In the presence of concurrent writes, we skip the last write phase. These writes leave the index writes in unverified status. Similarly, if the first or second phase write fails, we do not proceed with the third phase"?

Reply:
Other tests cover those cases